stress/frontend/src/main/scala/com/iheart/kanaloa/stress/http/StressHttpFrontend.scala

```diff
@@ -0,0 +1,76 @@
+/*
+* Copyright [2016] [iHeartMedia Inc]
+* All rights reserved
+*/
+package com.iheart.kanaloa.stress.http
+
+import akka.actor.{ActorRef, ActorSystem, Props}
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Directives._
+import akka.stream.ActorMaterializer
+import com.iheart.kanaloa.stress.backend.MockBackend
+import akka.pattern.{AskTimeoutException, ask}
+import akka.util.Timeout
+import scala.util.{Failure, Success}
+import scala.io.StdIn._
+import java.io.File
+import com.typesafe.config.ConfigFactory
+import kanaloa.reactive.dispatcher.PushingDispatcher
+import scala.concurrent.duration._
+
+object StressHttpFrontend extends App {
+  implicit val system = ActorSystem("Stress-Tests")
+  implicit val materializer = ActorMaterializer()
+  implicit val execCtx = system.dispatcher
+  implicit val timeout = Timeout(100.seconds)
+
+  val cfg = ConfigFactory.parseFile(
+    new File("./stress/frontend/config/stressTestInfra.conf")
+  )
+
+  val backend = system.actorOf(
+    Props(new MockBackend.BackendRouter(
+      cfg.getInt("sweetSpot"),
+      cfg.getInt("minRate")
+    )),
+    name = "backend"
+  )
+
+  val dispatcher =
+    system.actorOf(PushingDispatcher.props(
+      name = "my-service1",
+      backend
+    ) {
+      case MockBackend.Message(msg) ⇒ Right(msg)
+      case _                        ⇒ Left("something fishy going on...")
```

Comment:
I'm basically not caring about anything else, should only get 1 type of msg there anyway. Will make more descriptive:

case anythingElse ⇒ Left("Dispatcher: MockBackend.Message() acceptable only. Recieved: "+anythingElse)